modules/phase_field/examples/anisotropic_interfaces/snow_adapt.i

```diff
@@ -0,0 +1,165 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 16
+  ny = 16
+  xmax = 9
+  ymax = 9
+  elem_type = QUAD4
+[]
+
+[Adaptivity]
+  initial_marker = marker
+  initial_steps = 3
+  max_h_level = 3
+  marker = marker
+  [./Markers]
+    [./marker]
+      type = ErrorFractionMarker
+      refine = 0.85
+      coarsen = 0.1
+      indicator = indicator
+    [../]
+  [../]
+  [./Indicators]
+    [./indicator]
+      type = GradientJumpIndicator
+      variable = w
+    [../]
+  [../]
+[]
+
+[Variables]
+  [./w]
+  [../]
+  [./T]
+    initial_condition = 0.0
```

Comment:
0.0 is the default, which you are also using for w, only you don't specify it there.